### PR TITLE
Feature/store

### PR DESCRIPTION
**Pull request by JbirdL86**
What's new :question: 

-     Add configureStore for redux store
-     Add book reducer actions and actions creators
-     Add functionality for adding books to the store
-     Add functionality for removing books from the store
-     Add redux-logger

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+
+  return (
+    <li className="book-item">
+      <BookHeader
+        bookId={bookId}
+        title={title}
+        author={author}
+      />
+      <p>{bookId}</p>
+      <BookStatus />
```
Comment (afanimartin) [inline src/components/BookItem.js:16]: Please implement `removeBookFromStore` button for deleting a book.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @JbirdL86, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookItem.js (hunk 1 above)
Comment (JbirdL86) [reply src/components/BookItem.js:16]: > Please implement `removeBookFromStore` button for deleting a book.

The remove button is in the BookHeader component, can you please take a second look?

Comment (henatan99) [reply src/components/BookItem.js:16]: This issue is well addressed! :clap:

**Review by henatan99 — APPROVED** [review]
Hi @JbirdL86 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**